src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (newRobot[robot[key]]) {
+      return null;
+    }
 
+    newRobot[robot[key]] = key;
```

Comment:
see the checklist, p.1

Reply:
@jstmpelowycz Thank you! Fixed the variables